specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualRouter.json

```diff
@@ -0,0 +1,699 @@
+          "type": "array",
+          "readOnly": false,
+          "description": "VirtualRouter IPs",
+          "items": {
+            "type": "string"
+          }
+        },
+        "hostedSubnet": {
+          "readOnly": false,
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "The Subnet on which VirtualRouter is hosted."
+        },
+        "hostedGateway": {
+          "readOnly": false,
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "The Gateway on which VirtualRouter is hosted."
+        },	
+        "peerings": {
+          "type": "array",
+          "readOnly": false,
+          "description": "List of references to VirtualRouterPeerings",
+          "items": {
+            "$ref": "./network.json#/definitions/SubResource"
+          }
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the resource.",
+          "$ref": "./network.json#/definitions/ProvisioningState"
+        }
+      },
+      "description": "Virtual Router definition"
+    },
+    "VirtualRouterPeering": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/VirtualRouterPeeringProperties",
+          "description": "The properties of the Virtual Router Peering."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets name of the peering unique to VirtualRouter. This name can be used to access the resource."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets a unique read-only string that changes whenever the resource is updated."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Rule Group type."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Virtual Router Peering resource"
+    },
+    "VirtualRouterPeeringProperties": {
+      "properties": {
+        "peerAsn": {
+          "type": "integer",
+		  "readOnly": false,
+          "format": "int64",
+          "minimum": 1,
+          "maximum": 4294967295,		  
+          "description": "peer ASN."
+        },
+        "peerIp": {
+          "type": "string",
+          "readOnly": false,
+          "description": "peer IP"
+        }, 
+        "provisioningState": {
+          "description": "The provisioning state of the resource.",
+          "$ref": "./network.json#/definitions/ProvisioningState"
+        }
+      },
+      "description": "Properties of the rule group."
+    },
+    "VirtualRouterListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VirtualRouter"
+          },
+          "description": "List of Virtual Routers in a resource group."
```

Comment:
>in a resource grou [](start = 50, length = 18)

nit - update description to not have resource group. as the List* call need not be resourcegroup level query..